### DISCUSSION TITLE
Ente Auth sends HTML-only emails for login verification

### DISCUSSION THREAD

**Discussion by SiteRelEnby** [ente-io/ente#1554] [Enhancements]
### Description

Massive pain if there is only a terminal-based or similar email client. Email should have both plain text and HTML parts.

Happy to fix myself if pointed in the right direction.

### What product are you using?

Ente Auth

### What platform are you using?

Mobile - Android, Desktop - Linux, Desktop - Windows

Not a good UX for the login workflow.
![image](https://github.com/ente-io/ente/assets/125829806/3a8ce464-e6e3-43c7-928d-2ac23bc9ad4d)

**Comment by vishnukvmd** [comment]
Hey, thanks for offering to help!

I'm not familiar with the protocol for appending a plain-text version via SMTP, but here's the code path for sending mails:
https://github.com/ente-io/ente/blob/e7ca7501684422fe666a3cad0dc692b8f4f23b67/server/pkg/utils/email/email.go#L32-L93

This function could optionally accept a `plainTextBody`.
